Makefile

```diff
@@ -69,18 +69,8 @@ e2e-wip:
 	cd tests/e2e && npm run kakunin -- --tags @wip && cd ../..
 
 e2e-install:
-	cd tests/e2e
-	npm install
-	node ./node_modules/protractor/bin/webdriver-manager update --gecko=false
-	cd step_definitions
```

Comment:
why removed?

Reply:
Because the files are in git, these symlinks weren't created properly, but we didn't notice yet, because they're already there: https://github.com/bolt/four/tree/master/tests/e2e/step_definitions